### ISSUE TITLE
Download link is broken. Is this project still supported?

### ISSUE THREAD

**Issue by ZiadJ** [artoolkit/ar6unity-wiki#10]
The download link for the Unity package(artoolkit-dist.s3.amazonaws.com/arunity6/6.0/ARUnity6.unitypackage) which was initially published here is no longer valid:
https://github.com/artoolkit/ar6unity-wiki/wiki/Setting-Up-Your-First-ARToolKit-for-Unity-Scene

Also the lack of response here and in the forums  makes me wonder if this project is still being developed. If that's not the case then it would help a lot if someone one working on it could show sign of life. Thanks.

**Comment by jakeparkernet** [comment]
Same here.

**Comment by philip-lamb** [comment]
No, as far as I know, ARToolKit  v6 is no longer being developed since August 2017. It is a DAQRI project and none of the creators of ARToolKit are involved with DAQRI anymore. Please join us at https://github.com/artoolkitx/ for the followon project from the original creators of ARToolKit.
